lib/cli/generators/EMBER/template/.storybook/config.js

```diff
@@ -1,9 +1,4 @@
-import { configure } from '@storybook/ember';
+import { load } from '@storybook/ember';
 
 // automatically import all files ending in *.stories.js
-const req = require.context('../stories', true, /\.stories\.js$/);
-function loadStories() {
-  req.keys().forEach(filename => req(filename));
-}
-
-configure(loadStories, module);
+load(require.context('../stories', true, /\.stories\.js$/), module);
```

Comment:
Shouldn't this api be 'configure', as per https://github.com/storybookjs/storybook/releases/tag/v5.2.0-beta.13 ?

Reply:
Yeah that `load` got renamed to `configure` in a later release